### PR TITLE
tests: follow up for vault definition

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

File: src/test/tests.utils.ts [hunk 1]
```diff
@@ -338,3 +341,12 @@ export async function mockBadgerSdk(
     provider: mockProvider,
   });
 }
+
+// mb use this function is jest.setup to mock all sdk inits for all chains
```
Comment (NamelsKing) [inline src/test/tests.utils.ts:345]: what do i think about this?

File: src/indexers/vault-definition-indexer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import { BadgerGraph, BadgerSDK, RegistryVault, VaultsService } from '@badger-dao/sdk';
+import * as gqlGenT from '@badger-dao/sdk/lib/graphql/generated/badger';
+import graphVaults from '@badger-dao/sdk-mocks/generated/ethereum/graph/loadSetts.json';
+import registryVaults from '@badger-dao/sdk-mocks/generated/ethereum/vaults/loadVaults.json';
+
+import { SUPPORTED_CHAINS } from '../chains/chain';
+import { mockSupportedChains, setupMapper, setupVaultsCoumpoundDDB } from '../test/tests.utils';
+import { captureVaultData } from './vault-definition-indexer';
+
+describe('captureVaultData task', () => {
```
Comment (axejintao) [inline src/indexers/vault-definition-indexer.spec.ts:11]: `vault-definition-indexer`

File: src/indexers/vault-definition-indexer.spec.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  const TIME_NOW_MS = 1660381887554;
```
Comment (axejintao) [inline src/indexers/vault-definition-indexer.spec.ts:12]: `TEST_CURRENT_TIMESTAMP`

File: src/indexers/vault-definition-indexer.spec.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      if (!graphVault) return { sett: null };
+
+      return { sett: <gqlGenT.SettQuery['sett']>graphVault, __typename: 'Query' };
+    });
+
+    await mockSupportedChains();
+
+    put = jest.spyOn(DataMapper.prototype, 'put').mockImplementation();
+  });
+
```
Comment (axejintao) [inline src/indexers/vault-definition-indexer.spec.ts:41]: describe('captureVaultData', () => tests)

**Review by axejintao — COMMENTED** [review]
lgtm, minor comments to keep testing library a bit more constant unless we can't use those values

**Review by axejintao — APPROVED** [review]

**Pull request merged**